lib/davinci_dtr_test_kit/payer_server_groups/payer_server_adaptive_response_search_validation_test.rb

```diff
@@ -34,9 +34,8 @@ class PayerAdaptiveFormResponseSearchTest < Inferno::Test
           next
         end
       end
-      raise tests_failed[profile_url][0] if !test_passed && !tests_failed[profile_url].blank?
+      raise assert test_passed, "No returned resources conform to the profile: http://hl7.org/fhir/us/davinci-dtr/StructureDefinition/dtr-questionnaire-adapt-search" if !test_passed && !tests_failed[profile_url].blank?
```

Comment:
1. Should this have a version (here and above) like you added to others? Seems like yes.
2. The combination or raise and assert looks wrong